src/hci/gnomex/controller/CreateAllDataTracks.java

```diff
@@ -189,18 +187,17 @@ public Command execute() throws RollBackCommandException {
     	  toCreate.add(labName);
     	  toCreate.add(analysisName);
     	      	  
-//    	  System.out.println("Creating directories");
```

Comment:
All this clean up looks great. Statements like this look a lot like debug statements and could be LOG.debug(...) so at least you can turn up the logging to seem then and then throttle it back to hide the output. Just for future reference.